### ISSUE TITLE
Sugerencia para una nueva opción 

### ISSUE THREAD

**Issue by JDavid444** [undefined06855/AutoCheckpoint#9]
Mi sugerencia es que podrías agregar una opción para que los checkpoints no se coloquen en absoluto. Mi caso es que me pasa que cuando estoy en un modo, por ejemplo en cubo, los checkpoints se colocan por si solos, y no quiero que eso suceda

**Comment by undefined06855** [comment]
so like the checkpoints don't get placed when you're in cube or something?

**Comment by JDavid444** [comment]
No, por ejemplo, yo tengo activada la opcion para que se coloque un checkpoint cuando cambia de vehículo, pero hay ocasiones en las que se colocan aunque no haya ningún portal de cambio de vehículo

**Comment by undefined06855** [comment]
can you get a video of this happening and a level id where it happens?

**Comment by JDavid444** [comment]
Claro.

https://github.com/undefined06855/AutoCheckpoint/assets/174282295/c9ee2dd7-e353-4cd6-a16d-b32cad23c091

Eso es una pequeña parte del nivel dry out, y como se ve, los checkpoints se colocan automáticamente, y yo solo quiero que se coloquen en un punto específico, como en un portal
